Modules/MUON/MID/src/DigitsQcCheck.cxx

```diff
@@ -159,6 +165,22 @@ Quality DigitsQcCheck::check(std::map<std::string, std::shared_ptr<MonitorObject
         mQualityMap[item.second->getName()] = qual;
       } // if mNTFInSeconds > 0.
     }
+    if (item.second->getName() == "NbLBEmpty") {
+      auto histo = static_cast<TH1F*>(item.second->getObject());
```

Comment:
why not use a dynamic cast and make sure that the result is not a nullptr ?

Reply:
In general, you could try to use 
```
      auto histo = dynamic_cast<TH1F*>(item.second->getObject());
      if (histo) {
             // do something with the plot
      } else { 
             // something went wrong
      }
```
I let you investigate this safer option and perhaps submit another PR as to not block this one. This could apply to all the static casts.